packages/extension/tailwind.config.ts

```diff
@@ -1,10 +1,11 @@
-/* eslint-disable @typescript-eslint/no-var-requires */
-const config = require('@dailydotdev/shared/tailwind.config');
+import type { Config } from 'tailwindcss';
+import config from '@dailydotdev/shared/tailwind.config';
 
 module.exports = {
   ...config,
   content: [
     './src/**/*.{js,ts,jsx,tsx}',
     './node_modules/@dailydotdev/shared/src/**/*.{js,ts,jsx,tsx}',
   ],
-};
+  // eslint-disable-next-line
```

Comment:
Our prettier doesn't find this TS valid (need to upgrade prettier, but will do later)

Reply:
Wohoo, we now have `satisfies` 👌